feature-utils/poly-look/src/react-components/fixedFooter.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+import "./fixedFooter.css";
+
+const FixedFooter = ({ children }) => {
+  return <div className="fixed-footer">{children}</div>;
```

Comment:
Since this seems to be essentially a div with a certain class name, would there be a way of having a generic component that you can instantiate with class names? I lost track of all the components, but is there a chance you could use the same pattern elsewhere?

Reply:
What do you mean by a generic component? Some `Container` component with a `div` and a list of classNames that we could chose?
 
This is something I asked @fhd last week, because I wasn't sure if adding a component or just a className in poly-look. He agreed in the idea of creating a new component (perhaps he changed his mind and thinks the generic component is a better idea, though 😅 ).

Another option is organising better the components that we have and group them in different folders.

Reply:
> What do you mean by a generic component? Some `Container` component with a `div` and a list of classNames that we could chose?

Yea, pretty much. Even without a closed list of class names. You chose one with CSS, you're good.
> 
> This is something I asked @fhd last week, because I wasn't sure if adding a component or just a className in poly-look. He agreed in the idea of creating a new component (perhaps he changed his mind and thinks the generic component is a better idea, though sweat_smile ).
> 

It was a suggestion; but it makes sense and will reduce the amount of boilerplate if you use this pattern repeatedly.

> Another option is organising better the components that we have and group them in different folders.

That will certainly help find duplications and DRYing, but it's a separate issue.

Reply:
I was wondering about the css class vs component as well since this is such a simple component, at least now I know 😂 
As for the generic `Container` I'm not so sure. We get rid of some duplication but we also loose the meaning/intention behind the component. In this case, we might as well get rid of the component(s) entirely and stick to css classes 🤷

Reply:
I have added the `gradient` as a boolean option to make the component a bit more versatile - for example, for the "Log out" that some other screens will have. It's still a simple component, but at least now it's not just a `div` and could be more reusable.